backend/users/.env.dev

```diff
@@ -0,0 +1,8 @@
+PORT=8881
+DB_HOST='localhost'
+DB_NAME='studioghibli'
+DB_USER='root'
+DB_PASSWORD='rootadmin'
```

Comment:
Try not to push pwds in dev

Reply:
it will help other developers to use the file. I have rename the file with .dev

Reply:
Okay.